macros/DefaultAPISidebar.ejs

```diff
@@ -45,7 +45,7 @@ if (webAPIGroups[0][group].overview) {
 }
 
 function buildSublist(pages, title) {
-  var result = '<li data-default-state="open"><a href="#"><strong>' + title + '</strong></a><ol>';
+  var result = '<li class="toggle"><details open><summary>' + title + '</summary><ol>';
```

Comment:
Here's the ``{{DefaultAPISidebar}}`` sidebar on https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events, it looks correct 👍 with minor whitespace changes:

<img width="718" alt="pr 789 defaultapisidebar" src="https://user-images.githubusercontent.com/286017/45329899-5689f080-b528-11e8-9694-6cfa3c41bc2c.png">